static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain-sections.js

```diff
@@ -0,0 +1,10 @@
+define(function () {
+    var sections = ['politics', 'world', 'business', 'commentisfree'];
+
+    function getSection(section) {
+        section = section.toLowerCase();
+        return /news/.test(section) || sections.indexOf(section) !== -1 ? 1 : 2;
```

Comment:
Is 1 and 2 good identifier for this? I would suggest we use something more descriptive: inisde-sections : outside-sections. Maybe that's not the best wording but you see the point. It will be more obvious in the outbrain-codes.js, especially as the logic is split into two modules.